packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -1513,6 +1553,29 @@ Expected input body:
 {"property": <don't include this property>}
 ```
 
+### Models_Property_Types_ModelNullableProperty_get
```

Comment:
Do those test belong in the Model Properties scenario and not their coresponding container
- Collection
- Dictionary

Reply:
I will keep the current position for these cases. Because the collection and dictionary cases are focusing on testing the response/request body as arrays or records. 

However in our nullable vaules we return normal models in which we contains some properties with nullable values.

Reply:
Maybe I am misunderstanding but from what I see is the array iteself contains nullable value and the dictionary itself contains nullable values not the property that is of type array/dictionary being null which is what this spec is meant to test.

Reply:
Got your point. I think we more likely request/response a payload in which its has a array property and in this container it may cover nullable item. I think in reality the more meaningful case would be the latter one. You could see the issue https://github.com/Azure/autorest.csharp/issues/3084.

```
alias nullableFloat = float32 | null;

model MyModel {
  listOfFloats?: nullableFloat[];   
}
```

This is why I design this case as property not array or dictionary. But we could add these cases in both position. How do you think?

Reply:
I totally understand that this is most likely what you’ll see in services but I wanted to keep things to the minimum required in Cadl ranch and in this case the mode property  bring no value.

op getNullableArray(): nullalbleFloat[] is i feel like what we really care to test here.

Reply:
Unless I’m missing something and there is something actually different about it being in a property

Reply:
Okay thanks for the explaination! I think it makes sense and I'll change them to collection/dictionary sections.

Reply:
I adjust the cases:

- Arrays/nullable-float -> NullableFloat[]
- Dictionary/nullable-float -> Record<NullableFloat>
- PropertyTypes/nullable-property -> ModelWithNullableProperty